resolwe/process/parser.py

```diff
@@ -111,9 +113,66 @@ def __init__(self, source):
         self.source = source
         self.processes = []
         self.base_classes = set()
+        # Mapping betwwen process slug and AST node representing the class.
+        self.ast_nodes = dict()
 
         super().__init__()
 
+    def _get_called_slug(self, process_slug, called_node: ast.Call):
+        """Get the slug of the process being called.
+
+        :raises RuntimeError: when slug could not be determined (for instance when
+            the slug is being a variable).
+        """
+        # Both methods (run_process and get_latest) take argument slug as the first
+        # argument, so no separation is necessary.
+        argument_names = ("slug", "inputs")
+        arguments = [arg for arg in called_node.args]
+        all_arguments = {
+            name: value for name, value in zip_longest(argument_names, arguments)
+        }
+        all_arguments.update(
+            {keyword.arg: keyword.value for keyword in called_node.keywords}
```

Comment:
Is there any benefit of retrieving all parameters? It looks like returning the first element would be sufficient for this function (actually, as pointed out in the comment above, checking if `slug` is a keyword argument and returning the first argument otherwise).

Reply:
You are right. I have modified the method.